src/testing/docker/test-standalone.sh

```diff
@@ -2,7 +2,8 @@
 set -o errexit -o nounset -o xtrace
 
 #### build image
-docker-compose build web
+docker-compose build
+docker-compose up -d
```

Comment:
For the same reason, can you please remove `docker-compose up` and build only the `web` container as before?
The line bellow already starts a container for the tests.